packages/mui-material-next/src/InputLabel/InputLabel.types.ts

```diff
@@ -59,7 +60,7 @@ export interface InputLabelOwnProps {
   /**
    * The variant to use.
    */
-  variant?: 'standard' | 'outlined' | 'filled';
+  variant?: 'outlined' | 'filled'; // TODO: standardize to TextFieldProps['variant']
```

Comment:
nit

```suggestion
  variant?: 'outlined' | 'filled'; // TODO v6: standardize to TextFieldProps['variant']
```

So we can search the entire codebase for `TODO v6`